token/js/src/extensions/group/state.ts

```diff
@@ -0,0 +1,33 @@
+import { struct, u32 } from '@solana/buffer-layout';
+import { publicKey } from '@solana/buffer-layout-utils';
+import { PublicKey } from '@solana/web3.js';
+import type { TokenGroup } from '@solana/spl-token-group';
+import type { Mint } from '../../state/mint.js';
+import { ExtensionType, getExtensionData } from '../extensionType.js';
+
+/** Buffer layout for de/serializing a TokenGroup extension */
+export const TokenGroupLayout = struct<TokenGroup>([
+    publicKey('updateAuthority'),
+    publicKey('mint'),
+    u32('size'),
+    u32('maxSize'),
+]);
```

Comment:
You don't need to redefine a layout here! The interface JS library supports serialization.

https://github.com/solana-labs/solana-program-library/blob/7e022ac1b2ac527d6607712521685e64d0ff50f8/token-group/js/src/state/tokenGroup.ts#L46-L61

Reply:
I created this for geting the `TOKEN_GROUP_SIZE` on the next line, not very familiar with all this, is there other ways to get the size?

Reply:
maybe `tokenGroupCodec.fixedSize`?